modules/encrypted_cos_bucket/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.49.0"
+      version = ">= 1.59.0"
```

Comment:
suggest to go with this for consistency with all other modules: `>= 1.59.0, <2.0.0`